app/plugin.go

```diff
@@ -496,7 +496,7 @@ func (a *App) getRemotePlugins(filter *model.MarketplacePluginFilter) ([]*model.
 		ServerVersion: model.CurrentVersion,
 	})
 	if err != nil {
-		return nil, model.NewAppError("GetMarketplacePlugins", "app.plugin.marketplace_plugins.app_error", nil, err.Error(), http.StatusInternalServerError)
+		return nil, model.NewAppError("GetMarketplacePlugins", "app.plugin.marketplace_server.app_error", nil, err.Error(), http.StatusInternalServerError)
```

Comment:
I'd perhaps suggest even more descriptive, benefitting the client-side: `app.plugin.marketplace_client.failed_to_fetch.app_error`.

(I have no real idea what our "schema" is for ids, but mimicking what I've seen elsewhere.)